config/settings.yml

```diff
@@ -342,30 +342,19 @@ discourse_ai:
     type: group_list
     list_type: compact
     default: "3|14" # 3: @staff, 14: @trust_level_4
-  # Adding a new bot? Make sure to create a user for it on the seed file and update translations.
   ai_bot_public_sharing_allowed_groups:
     client: false
     type: group_list
     list_type: compact
     default: "1|2" # 1: admins, 2: moderators
     allow_any: false
     refresh: true
-  ai_bot_enabled_chat_bots:
+  ai_bot_enabled_chat_bots: # TODO(roman): Make this dynamic
     type: list
-    default: "gpt-3.5-turbo"
+    default: ""
```

Comment:
Unsetting the default means some instances will have this feature disabled after a deploy. I guess you're working on a migration for that too, right? (Not sure I'm reading the internal topic correctly.)

Reply:
We track enabled models in the `llm_models` table. I made this hidden and update the migration to copy the data.